moler/cmd/unix/genericunix.py

```diff
@@ -126,22 +127,36 @@ def is_end_of_cmd_output(self, line: str) -> bool:
         :param line: Line from device.
         :return: True if end of command is reached, False otherwise.
         """
-        if not self._kill_ctrl_z_sent and self._ctrl_z_sent:
+        if not self._kill_ctrl_z_job_done and self._ctrl_z_sent:
             return False
         return super(GenericUnixCommand, self).is_end_of_cmd_output(line=line)
 
     # [2]+  Stopped
     _re_ctrl_z_stopped = re.compile(r"\[(?P<JOB_ID>\d+)\]\+\s+Stopped")
 
+    # [2]+  Done
+    _re_kill_done = re.compile(r"\[(?P<JOB_ID>\d+)\]\+\s+Done")
```

Comment:
Do you use group JOB_ID from this regex in any place?

Reply:
You are right, group is unnecessary. Matching is a point here. Corrected.